workflow/executor/docker/docker.go

```diff
@@ -65,7 +65,9 @@ func (d *DockerExecutor) CopyFile(containerID string, sourcePath string, destPat
 		return err
 	}
 	if !file.ExistsInTar(sourcePath, tar.NewReader(gzipReader)) {
-		return errors.InternalErrorf("path %s does not exist (or %s is empty) in archive %s", sourcePath, sourcePath, destPath)
+		errMsg := fmt.Sprintf("path %s does not exist (or %s is empty) in archive %s", sourcePath, sourcePath, destPath)
```

Comment:
I think the step should not fail if the file exists but empty. Imagine there is an output which holds corrupted lined of a table. Being empty is its normal state. The workflow should not fail in that case.

Reply:
I agree that an empty file is a valid case. I believe the error is misleading and file.ExistsInTar is checking the right thing.

Reply:
>I believe the error is misleading and file.ExistsInTar is checking the right thing.

This does not seem to be the case. The reason I'm writing is that my pods are failing when output files are empty, which is very frustrating and surprising (I've created component that can write either to GCS or BigQuery and has two outputs where one is always supposed to be empty).